odoo/addons/base/res/res_config.py

```diff
@@ -492,6 +506,40 @@ def default_get(self, fields):
             if self._fields[name].type == 'selection':
                 res[name] = int(res[name])
 
+        # config: get & convert stored ir.config_parameter (or default)
+        for name, icp in classified['config']:
+            field = self._fields[name]
+            value = SuIrConfigParameter.get_param(icp, field.default(self) if field.default else False)
+            if value is not False:
+                if field.type in ['many2one']:
```

Comment:
`in` is overkill, `== 'many2one'` works just as well

Reply:
indeed